docker/hardhat/hardhat.config.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
                 accountsBalance: "1000000000000000000000000"
             },
             gas: 1e7,
-            gasPrice: 1
+            gasPrice: 1,
+            chainId: 1
```

Comment:
I put this in so that when running against a forked node the correct contract addresses are loaded in e.g. by running `BalancerV2Vault::deployed(web3).await.unwrap()`.

I've tested locally and it doesn't break the existing tests AFAIK, but looking forward to seeing CI run against this.

Reply:
Hmm, this would be an issue if somebody's solver only works on some other chain but still wants to use a forked node. But I guess we can figure that out when the first person is interested in that.

Reply:
Yeah, I hear you on that. Unfortunately hardhat doesn't support changing the `chainId` once the process is running, I already checked if I could do that to avoid changing this file at all. We could add different networks though and invoke the correct network via docker compose. Or, maybe there is an even better solution we could come to once needed.